crates/era-cheatcodes/tests/src/cheatcodes/Fork.t.sol

```diff
@@ -78,4 +79,44 @@ contract ForkTest is Test {
 
         require(activeFork == forkId, "Active fork is not correct");
     }
+
+    /// checks that marking as persistent works
+    function testMarkPersistent() public {
+        require(vm.isPersistent(address(this)) == true, "should be persistent");
+
+        // the dummy address should not be persistent
+        require(
+            vm.isPersistent(DUMMY_ADDRESS) == false,
+            "should not be persistent"
+        );
+
+        // mark the dummy address as persistent
+        vm.makePersistent(DUMMY_ADDRESS);
```

Comment:
Could we check it together with `selectFork` that we are actually doing everything correct